src/utils/index.test.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from '@jest/globals';
+import { BigNumber as BNJS } from 'bignumber.js';
+
+import {
+  commifyString,
+  formatRelays,
+  formatFillPercentage,
+  formatDaysFromTimestamp,
+  formatOwnershipShare,
+  parseInputValue,
+  shortenAddress,
+} from 'utils';
+
+describe('Formatting functions', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should commify strings', () => {
+    const commified = commifyString('123456789');
+    expect(commified).toEqual('123,456,789');
+  });
+
+  it('should format BigNumber relays', () => {
+    const relays = formatRelays(new BNJS(100000000));
+    expect(relays).toEqual('100.00');
+  });
+
+  it('should format string relays', () => {
+    const relays = formatRelays('100000000');
+    expect(relays).toEqual('100.00');
+  });
+
+  it('should format days left as a percentage', () => {
+    const fillPercentage = formatFillPercentage(500, 5000);
+    expect(fillPercentage).toEqual(90);
```

Comment:
Could we just add a few more tests for these? Mostly for edge cases, like very big or very small numbers :)

Reply:
Fixed in 369bea50965c69257b96988549e13de5abb02769